docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -462,8 +466,35 @@
~}
~----
 <1> By making the class a managed bean, Quarkus will take the customizer into account when it starts the Vert.x servers
 <2> In this case, we only care about customizing the HTTP server, so we just override the `customizeHttpServer` method, but users should be aware that `HttpServerOptionsCustomizer` allows configuring the HTTPS and Domain Socket servers as well
```

Comment:
I don't think we really link to examples that are not dependencies of the project itself.

Reply:
So you want me to remove the link right? Or would you like me to include the third party library 😄

Reply:
And do you think we should provide some kind of reference or code example on how the end-user can enable hot reloading or other ssl options?

Reply:
I added the code snippet for reloading ssl which I had mentioned earlier here: https://github.com/quarkusio/quarkus/issues/15926#issuecomment-1545002471 What do you think of it, are you ok or would you like me to adjust it?

Reply:
cc @cescoffier

Reply:
I would avoid relying on a 3rd party library. It should be possible to implement SSL reload without depending on anything.

Reply:
I am the maintainer of the library. I understand you want to avoid any 3rd party library to make Quarkus less dependent and keep the performance as high as possible. Then I would suggest to close this PR as adding documentation does not add that much of value for explaining the end-user how to customize their ssl configuration. It might be then better to have a new PR with the actual code change to have ssl/tls reloading within Quarkus itself.

Shall I close this PR then? What do you think @geoand and @cescoffier how should we proceed?

Reply:
> It might be then better to have a new PR with the actual code change to have ssl/tls reloading within Quarkus itself.

We could do it in this PR, no?

Reply:
Sure, I will give it a try

Reply:
Hi guys, it has been awhile since I created the pr. I am just being honest that I am not that motivated anymore to make the code changes as it feels like I will be copying parts of my library which has similar functionalities to this project. I think it is better if someone else can work on this feature